File: 2st/Riyeon/배열 만들기 2.py
def solution(l, r):
    answer = []
    for i in range(l, r+1):
        s = str(i).replace('0', '')
        s = s.replace('5', '')
        if s == '':
            answer.append(i)
    return answer if len(answer) != 0 else [-1]

File: 4st/Eunseong/배열 만들기 2.java
import java.util.ArrayList;

class Solution {
    public int[] solution(int l, int r) {
        ArrayList<Integer> arr = new ArrayList<>();
        for (int i = 1; Integer.valueOf(Integer.toBinaryString(i).replace("1", "5")) <= r; i++) {
            if (Integer.valueOf(Integer.toBinaryString(i).replace("1", "5")) >= l) {
                arr.add(Integer.valueOf(Integer.toBinaryString(i).replace("1", "5")));
            }
        }
        if (arr.size() == 0) {
            int answer[] = {-1};
            return answer;
        }
        else {
            int answer[] = new int[arr.size()];
            for (int i = 0; i < arr.size(); i++) {
                answer[i] = arr.get(i);
            }
            return answer;
        }
    }
}